### PR TITLE
migrate alpha feaures landing page

### PR DESCRIPTION
**Pull request by christinaausley**
## Description

Relates to https://github.com/camunda/camunda-docs/pull/4213#discussion_r1800581137. Trials moving the alpha features page to components.

## When should this change go live?

- [ ] This is a bug fix, security concern, or something that needs **urgent release support**.
- [ ] This is already available but undocumented and should be released within a week.
- [ ] This on a **specific schedule** and the assignee will coordinate a release with the DevEx team. (apply `hold` label or convert to draft PR)
- [x] This is part of a scheduled alpha or minor. (apply alpha or minor label)
- [ ] There is **no urgency** with this change and can be released at any time.

## PR Checklist

- [ ] My changes are for an **already released minor** and are in `/versioned_docs` directory.
- [x] My changes are for the **next minor** and are in `/docs` directory (aka `/next/`).

- [ ] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned the Engineering DRI or delegate.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @camunda/tech-writers as a reviewer.

### CODE REVIEW

**Comment by christinaausley** [comment]
Just need to add a redirect.

File: docs/components/early-access/overview.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ These features may be unstable, and subject to significant changes or removal.
 
 ## Alpha features
 
-[Alpha features](/reference/alpha-features.md) are more developed and closer to becoming part of the product but may not yet be fully optimized or supported. Selected Camunda features and components are released as alpha versions to provide early access. By testing these features, you have the opportunity to participate in their development by sharing feedback before they reach General Availability (GA).
+[Alpha features](/components/early-access/alpha/alpha-features.md) are more developed and closer to becoming part of the product but may not yet be fully optimized or supported. Selected Camunda features and components are released as alpha versions to provide early access. By testing these features, you have the opportunity to participate in their development by sharing feedback before they reach General Availability (GA).
```
Comment (conceptualshark) [inline docs/components/early-access/overview.md:23]: Do we want to link "General Availability (GA)" here to that section of the alpha features doc, like it is on the other page?

**Review by conceptualshark — APPROVED** [review]
This looks like it's in a good state to me, with just one comment where another link might be helpful. 

- Is it worth taking the "General Availability" section out of the Alpha features page, and instead adding it to the [Release policy](https://docs.camunda.io/docs/reference/release-policy/) page? It seems like it could better serve as a standing definition for reference, as opposed to being mentioned only within /components with the rest of this content.

- Possibly for a discussion outside of this PR: I didn't immediately assume the overview pages were going to be part of the top-level category in the sidebar.  The Components section defaults to using a separate overview or introductory page, generally, but I don't think we have consistency for this assumption across the docs, so it's not something I think needs to change here - but we may want to visit this behavior more broadly (I've put it on our sync notes!).

File: docs/components/early-access/overview.md (hunk 1 above)
Comment (christinaausley) [reply docs/components/early-access/overview.md]: Done 👍

File: static/.htaccess [hunk 2]
```diff
@@ -17,6 +17,10 @@ RewriteRule ^optimize/3.14.0/(.*)$ /optimize/$1 [R=302,L]
 # 8.6 content moves vvvv
 # ----------------------
 
+# Move alpha features to components
+RewriteRule ^docs/next/reference/alpha-features/?$ /docs/next/components/early-access/alpha/alpha-features/$1 [R=301,L]
+RewriteRule ^docs/reference/alpha-features/?$ /docs/components/early-access/alpha/alpha-features/$1 [R=301,L]
```
Comment (akeller) [inline static/.htaccess:22]:
```suggestion
```

**Review by akeller — APPROVED** [review]
Nothing blocking with this PR, but I would like a follow up based on @conceptualshark's feedback. I think there is some work to be done, but could happen in this PR or another. 

> Is it worth taking the "General Availability" section out of the Alpha features page, and instead adding it to the [Release policy](https://docs.camunda.io/docs/reference/release-policy/) page? It seems like it could better serve as a standing definition for reference, as opposed to being mentioned only within /components with the rest of this content.

**Comment by akeller** [comment]
Ha, another PR it is then 😅

**Comment by christinaausley** [comment]
> Is it worth taking the "General Availability" section out of the Alpha features page, and instead adding it to the [Release policy](https://docs.camunda.io/docs/reference/release-policy/) page? It seems like it could better serve as a standing definition for reference, as opposed to being mentioned only within /components with the rest of this content.

Whoops -- auto-merge got us. I'll open a separate issue.

**Pull request merged**